ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -316,14 +316,24 @@ def _wan_load_balance_policies(self) -> list:
                     context_keys=["name"],
                 )
 
-            default_virtual_topology = get(policy, "default_virtual_topology", required=True)
+            default_virtual_topology = get(
+                policy,
+                "default_virtual_topology",
+                required=True,
+                org_key=(
+                    f"A 'default_virtual_topology must be defined for policy {policy['name']}. "
+                    "It is possible to disable default-match by setting 'drop_unmatched' to 'true'."
+                ),
+            )
             if not get(default_virtual_topology, "drop_unmatched", default=False):
                 name = get(default_virtual_topology, "name", default=f"{policy['name']}-DEFAULT")
                 context_path = f"wan_virtual_topologies.policies[{policy['name']}].default_virtual_topology"
                 append_if_not_duplicate(
                     list_of_dicts=wan_load_balance_policies,
                     primary_key="name",
-                    new_dict=self._generate_wan_load_balance_policy(f"LB-{name}", default_virtual_topology, context_path),
+                    new_dict=self._generate_wan_load_balance_policy(
```

Comment:
Why not just do the _default_all check in the function you call instead of here where it seems a bit "clunky"?

Reply:
hmm the mechanism using input was already used some other place for control_plane LB policy and here I am reusing this but I can just use the key instead and change the behavior. Let me refactor to make it less clunky